packages/lab/src/static-list/StaticListItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { makePrefixer } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
+import { type ComponentPropsWithoutRef, forwardRef } from "react";
+
+import staticListItemCss from "./StaticListItem.css";
+
+const withBaseName = makePrefixer("saltStaticListItem");
+
+export interface StaticListItemProps extends ComponentPropsWithoutRef<"li"> {
+  /**
+   *  To pass a divider to separate the List items. Defaults to false.
+   */
+  divider?: boolean;
+}
+
+export const StaticListItem = forwardRef<HTMLLIElement, StaticListItemProps>(
+  function TrackerStep(props, ref) {
+    const { className, children, divider = false, ...restProps } = props;
+
+    const targetWindow = useWindow();
+    useComponentCssInjection({
+      testId: "salt-static-list-item",
+      css: staticListItemCss,
+      window: targetWindow,
+    });
+
+    return (
+      <li
+        className={clsx(
+          withBaseName(),
+          {
+            [withBaseName("divided")]: divider,
```

Comment:
Is there a reason why we could not use `children` and composition ?
You could have different types of divider then.

NIT: naming does not align but hopefully we won't need this code, if we go through the composition route?

Reply:
We can use composition. 
Was following what we have for list. The OptionGroup component passes border internally. 

But in our case since this is optional, will update this use-case using composition.